flink-python/pyflink/table/table_environment.py

```diff
@@ -539,6 +571,7 @@ def use_catalog(self, catalog_name):
         +----------------+-----------------------------------------+
 
         :param: catalog_name: The name of the catalog to set as the current default catalog.
```

Comment:
Need to remove the ':' after param, or else the param str will not be parsed by sphix

Reply:
Good catch!